### PR TITLE
Add installation guide for Windows users

### PR DESCRIPTION
**Pull request by aaronsteers**
### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request enhances existing functionality.
- [ ] This pull request introduces breaking change.

For more information, see the [Contributing Guide](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Adds guidance for installing via Windows, using the Chocolatey package manager.

### Issues Resolved

n/a

### Checklist

Put an `x` into all boxes that apply:

- [x] I have read the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

#### Tests

- [ ] I have added tests to cover my changes.
- [ ] All tests pass when I run `make test`.

#### Documentation

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

#### Code Style

- [x] My code follows the code style of this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ If you are a Mac OS X user, you can use [Homebrew](https://brew.sh):
 brew install terraform-docs
 ```
 
+Windows users can install using [chocolatey](https://www.chocolatey.org):
+
+```
+choco install terraform-docs
```
Comment (khos2ow) [inline README.md]: I'm not familiar with Chocolatey, do we need to publish the binary to their system for this to work (something like brew)?

Comment (aaronsteers) [reply README.md]: Yes, it works very similar to brew. And it looks like someone ("[DrFaust](https://chocolatey.org/profiles/DrFaust)"?) has already created a wrapper around the 8.2 installer in the releases folder here in github. (You can see their script in the "Files" section on the link below.) 

https://chocolatey.org/packages/Terraform-Docs

Comment (aaronsteers) [reply README.md]: Update: I was able to trace the package source code using the link on the Chocolatey website. The manifest file is a `.nuspec` file, and then there are two powershell scripts that do the remainder of the work. The author is @drfaust92 here on Github.

https://github.com/DrFaust92/chocolatey-packages/tree/master/terraform-docs

Comment (DrFaust92) [reply README.md]: Hey guys, I've added this package to choco(and closed my issue as I didn't notice this one)

I handled the the installation script, this is indeed like brew but for Windows.

Ill keep updating this as new versions are released.

Comment (khos2ow) [reply README.md]: Thanks @aaronsteers for the update.
@DrFaust92 that is awesome. How do you feel to move the config you've been maintaining to this repo (as an official config) e.g. in `scripts/choco/` and keep maintaining it here (or may be we can even automate it on release)

File: README.md [hunk 2]
```diff
@@ -34,6 +34,12 @@ If you are a Mac OS X user, you can use [Homebrew](https://brew.sh):
 brew install terraform-docs
 ```
 
+Windows users can install using [chocolatey](https://www.chocolatey.org):
```
Comment (khos2ow) [inline README.md]: Also one minor item, can you change the link to `Chocolatey` since it's a name.

File: README.md (hunk 1 above)
Comment (DrFaust92) [reply README.md]: Sure, I've been looking into automating this either way.
I'll open a seperate PR after I think how should the access key to Choco can be handled.

File: README.md (hunk 2 above)
Comment (aaronsteers) [reply README.md]: Good catch, thanks! This is complete.

File: README.md (hunk 1 above)
Comment (khos2ow) [reply README.md]: That's great. We can track that in another issue/PR of its own. I'm gonna merge this. Thanks folks.

**Pull request merged**